Maelabord.Rmd

```diff
@@ -0,0 +1,270 @@
+---
+title: "Mælabord"
+author: "The North"
+date: "2024-11-06"
+output: html_document
+---
+
+

```

Comment:
/// suggestion
# Tengist SQLite gagnagrunninum
conn <- dbConnect(RSQLite::SQLite(), "C:/Users/Lenovo/Documents/GitHub/capstone-the-north/data/rotten_tomatoes.db")

# Skoða hvaða töflur eru í gagnagrunninum
dbListTables(conn)
///

Síðan setja inn alla gagnagrunna með Heiti_tafla <- dbReadTable(con, "heitið á töflunni í db")